SPECS/libcap-ng/libcap-ng.spec

```
@@ -59,14 +74,16 @@ find %{buildroot} -type f -name "*.la" -delete -print
 
 %files devel
 %defattr(-, root, root)
-%{_libdir}/*.so
+%{_libdir}/*.so*
```

This seems to have been correct - we don't need the actual library in the `devel` subpackage. It is common practice for the default subpackage to provide the actual library, while `devel` contains links. And since `devel` depends on the default subpackage, you always have the actual library available during build-time, while everything can build with `-l<libname>` where the linker will look for `<libname>.so`.
```suggestion
%{_libdir}/*.so
```

---

I added it because I got this error in my logs:

Installed (but unpackaged) file(s) found:"
time="2024-04-11T01:30:26Z" level=debug msg="   /usr/lib/libcap-ng.so.0"
time="2024-04-11T01:30:26Z" level=debug msg="   /usr/lib/libcap-ng.so.0.0.0"
time="2024-04-11T01:30:26Z" level=debug msg="   /usr/lib/libdrop_ambient.so.0"
time="2024-04-11T01:30:26Z" level=debug msg="   /usr/lib/libdrop_ambient.so.0.0.0"

Do you think it's better to move it up to %files or be more exact with filenames maybe?

---

Yeah, it looks like after I removed the "rm" files lines everything came back to normal!
Buddy build [550835](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=550835&view=results)